project-management/src/main/java/life/qbic/projectmanagement/application/api/PurchaseStoreException.java

```diff
@@ -0,0 +1,38 @@
+package life.qbic.projectmanagement.application.api;
+
+import java.io.Serial;
+
+/**
+ * <b>Purchase Store Exception</b>
+ * <p>
+ * Exception that shall be thrown if a
+ * {@link life.qbic.projectmanagement.domain.model.project.purchase.ServicePurchase} item cannot be
+ * stored in the persistence storage implementation.
+ *
+ * @since 1.0.0
+ */
+public class PurchaseStoreException extends RuntimeException {
```

Comment:
As you are only catching this exception to log and rethrow you could just not catch the exception and let the global exception handler deal with it. You would not need this special exception then.

Reply:
I object to promote explicit language against general technical ones :)